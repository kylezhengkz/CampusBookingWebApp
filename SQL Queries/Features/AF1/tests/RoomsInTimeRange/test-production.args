{
    "args": [
        "00000000-0000-0000-0000-0000000001e8",
        "2025-01-17 08:00:05",
        "2025-12-17 08:00:05",
        null
    ],
    "kwargs": {}
}   