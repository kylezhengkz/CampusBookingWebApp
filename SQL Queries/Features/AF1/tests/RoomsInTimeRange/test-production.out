{'roomID': UUID('00000000-0000-0000-0000-00000000006c'), 'roomName': 'ESB2 1', 'bookingCount': 2}
{'roomID': UUID('00000000-0000-0000-0000-0000000000d3'), 'roomName': 'MASB1 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000013e'), 'roomName': 'SLH1 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000172'), 'roomName': 'CLEB2 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000001b6'), 'roomName': 'MHLH1 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000243'), 'roomName': 'PEB2 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000286'), 'roomName': 'DWH1 2', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000002cd'), 'roomName': 'HC2 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000002d8'), 'roomName': 'DWH3 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000304'), 'roomName': 'AA1 2', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000003aa'), 'roomName': 'CMH5 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000003f5'), 'roomName': 'AB4 2', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000004b1'), 'roomName': 'FL4 2', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000052e'), 'roomName': 'PC4 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000567'), 'roomName': 'MASB5 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000005c9'), 'roomName': 'EC1 5', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000060d'), 'roomName': 'AC2 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000613'), 'roomName': 'AR1 5', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000643'), 'roomName': 'CMH3 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000702'), 'roomName': 'CMWB2 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000807'), 'roomName': 'VWLH1 1', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000081f'), 'roomName': 'HSB4 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000899'), 'roomName': 'FEB5 6', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000095b'), 'roomName': 'EV1 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000009b6'), 'roomName': 'CLEB5 2', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000abf'), 'roomName': 'CMNB2 2', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000c57'), 'roomName': 'AV1 9', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000000d15'), 'roomName': 'CMSB1 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000001016'), 'roomName': 'HEB2 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000010f2'), 'roomName': 'CAPNB8 6', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000001116'), 'roomName': 'HLH4 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000011cf'), 'roomName': 'CEB3 6', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-0000000011ec'), 'roomName': 'DPB6 4', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000136a'), 'roomName': 'MA7 8', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-000000001371'), 'roomName': 'JAC2 15', 'bookingCount': 1}